packages/@monorepo-utils/workspaces-to-typescript-project-references/src/manager/workspaces.ts

```diff
@@ -26,7 +26,7 @@ export const plugin: PackageManagerPlugin = (options) => {
         resolve({ name, version }): string | null {
             const matchPkg = monorepoPackages.find((info) => {
                 if (info.packageJSON.name !== name) return false;
-                return !!semverMatch(version, [info.packageJSON.version]);
+                return !!semverMatch(version, [info.packageJSON.version]) || version.startsWith("workspace:");
```

Comment:
```js
if (version.startsWith("workspace:")){
   return true
}
return !!semverMatch(version, [info.packageJSON.version]);
```

It can be more readable.

Reply:
Thanks for the quick response! I went with the one-liner `if` to be consistent with previous line.